package.json

```diff
@@ -38,18 +34,20 @@
     "css-loader": "^1.0.0",
     "express": "^4.16.2",
     "file-loader": "^3.0.1",
-    "gts": "^0.8.0",
-    "hadouken-js-adapter": "^0.39.1",
     "node-fetch": "^2.3.0",
     "style-loader": "^0.21.0",
     "ts-jest": "^23.10.4",
     "ts-loader": "^4.4.2",
     "tslint-react": "^3.6.0",
-    "typescript": "^3.1.1",
     "url-loader": "^1.0.1",
     "webpack": "^4.16.1",
     "webpack-cli": "^3.0.8",
-    "webpack-dev-middleware": "^3.6.0"
+    "webpack-dev-middleware": "^3.6.0",
+    "eslint": "^5.15.3",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-react": "^7.12.4",
+    "@typescript-eslint/eslint-plugin": "^1.5.0",
+    "@typescript-eslint/parser": "^1.5.0"
```

Comment:
Can we keep this list sorted, to prevent unnecessary changes/conflicts when next adding or removing a package?

You can [apparently](https://stackoverflow.com/a/45187142) get NPM to do this for you - just run `npm remove -S example; npm remove -D example`

Reply:
Fixed